.github/workflows/alpha-release.yml

```diff
@@ -0,0 +1,81 @@
+name: Manual Alpha Package Release
+
+on:
+  # triggering in automated way to appear in github workflows
+  pull_request:
+    branches: [main]
+
+  # Allows running workflow manually from the GitHub Actions tab
+  workflow_dispatch:
+
+jobs:
+  build_and_test:
+    name: Build and Test
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE so job can access it
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      # # Ensure node version is great enough
+      # - name: Use Node.js v12.x
+      #   uses: actions/setup-node@v1
+      #   with:
+      #     node-version: '12.x'
+
+      # # Install dependencies
+      # - name: Install rush
+      #   run: npm install -g @microsoft/rush
+      # - name: Install dependencies
+      #   run: rush install
+
+      #   # Get datetime
+      # - name: Get datetime for alpha release name
+      #   id: datetime
+      #   run: |
+      #     echo "::set-output name=datetime::$(date +'%Y%m%d.%H%M')"
+
+      # # Bump alpha package versions
+      # - name: Bump alpha package versions
+      #   run: node common/config/node_modules/beachball/bin/beachball canary --canary-name alpha-${{ steps.datetime.outputs.datetime }} --tag dev --no-publish
+
+      # # Build packages
+      # - name: Build @azure/communication-react package
+      #   run: rush build -t @azure/communication-react
+
+      # # Test Packages
+      # - name: Test @azure/communication-react package
+      #   run: rush test -t @azure/communication-react
+
+      # # Push git tags
+      # - name: Create and push git tags
+      #   run: |
+      #     git tag 1.0.0-alpha-${{ steps.datetime.outputs.datetime }}
+      #     git push --tags
+
+      # # Publish package
+      # - name: Package alpha packages for release
+      #   run: npm pack
+      #   working-directory: ./packages/communication-react
+
+      # Deploy npm package - this is done by uploading to Azure's SDK blob storage then triggering their partner release pipeline.
+      # More info: https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/1/Partner-Release-Pipeline
+      #
+      # - name: Upload tarball to blob storage
+      #   uses: azure/CLI@v1
+      #   with:
+      #     inlineScript: |
+      #       az login --service-principal -u cfb5fe52-8bb2-4901-9056-70449fa2df96 -p ${{ secrets.AZURESDKPARTNERDROPS_SERVICE_PRINCIPAL_KEY }} --tenant 72f988bf-86f1-41af-91ab-2d7cd011db47
+      #       az extension add --name storage-preview
+      #       az storage azcopy blob upload -s "packages/communication-react/release/*" -c "drops/azure-communication-services/react/npm/1.0.0-alpha-${{ steps.datetime.outputs.datetime }}" --account-name azuresdkpartnerdrops
+      #
+      # Likely blocked on: https://github.com/Azure/pipelines/pull/11
+      # blob uploaded for testing alpha release: 1.0.0-alpha-20210601.2201
+      - uses: Azure/pipelines@v1
```

Comment:
Currently not working with:
![image](https://user-images.githubusercontent.com/2684369/120525553-15e3fc80-c38d-11eb-9763-49ffb352a7c4.png)

Reply:
(also will likely need https://github.com/Azure/pipelines/pull/11 even if we get passed this error)

Reply:
Fixed the first issue, in debugging the action, where the action does `GetDefinitions` the api call is actually an `matches that contain` search so is returning both `azuresdkpartnerdrops to npm` and `azuresdkpartnerdrops to npm - Copy`.

Instead it should be matching the exact name provided in the action. I will create a PR with the fix but given the lack of traction on issues in that repo we will likely need to use a fork of it. I have pinged the repo owners internally on the second issue about providing pipeline variables on Teams but no response yet.

Reply:
Woohoo ran locally with the variables fix and the build definition fix and works!
Release: https://dev.azure.com/azure-sdk/internal/_releaseProgress?_a=release-pipeline-progress&releaseId=18251
On npm: https://www.npmjs.com/package/@azure/communication-react/v/1.0.0-alpha-20210601.2201.0

Reply:
Note: submitted PR to original repo with fix for searching for build definitions: https://github.com/Azure/pipelines/pull/34